app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -928,6 +940,7 @@ public String toJSON() {
             jsonObject.put("raw_calculated", raw_calculated);
             jsonObject.put("raw_data", raw_data);
             jsonObject.put("calculated_value_slope", calculated_value_slope);
+            jsonObject.put("calibration_uuid", calibration_uuid);
```

Comment:
The `estimated_bg` method is only used by the depreciated and broken "predictive" mode. This setting is only accessible via engineering mode and is checked and warned for by `checkBadSettings()` if anyone were to enable it. Eventually I think it should be removed completely. I don't think that is needed, especially not on the follower. 

The other usages of a,b,c are just when calculating curves for calibration which should only happen on the master. What do you think? I reckon it should be possible to drop them unless I have missed something but I did look at this before when I was trying to work out exactly what a,b,c even were for.  I can test removing these tonight to see whether there is any visible impact on the follower.

Reducing the size of them might be an idea but I am not certain what the implications of that might be. 

Your other suggestion of only adding the calibration uuid on single entries might also work as a good alternative. I would have to look at the code flow as to whether we know this ahead of time. For example when batching the entries due to rate limit (which would normally only happen with an nfc based sensor)